lib/Dialect/AIE/IR/AIETargetModel.cpp

```diff
@@ -291,9 +291,8 @@ bool AIE1TargetModel::isLegalTileConnection(int col, int row,
 }
 
 std::vector<std::pair<uint32_t, uint32_t>>
-AIE1TargetModel::getBurstEncodingsAndLengths() const {
-  return {std::pair(0x00000000, 64), std::pair(0x40000000, 128),
-          std::pair(0x80000000, 256)};
+AIE1TargetModel::getShimBurstEncodingsAndLengths() const {
+  return {std::pair(0, 64), std::pair(1, 128), std::pair(2, 256)};
```

Comment:
The encoding would not have been correct for AIE1 although no code path for AIE1 uses it.